LoRaEngine/deployment.ci.aio.template.json

```diff
@@ -184,6 +184,31 @@
             "status": "running",
             "restartPolicy": "always",
             "version": "1.0"
+          },
+          "metricscollector": {
+            "settings": {
+              "image": "mcr.microsoft.com/azureiotedge-metrics-collector:1.0"
+            },
+            "type": "docker",
+            "env": {
+              "ResourceId": {
+                "value": "$IOT_HUB_RESOURCE_ID"
+              },
+              "UploadTarget": {
+                "value": "AzureMonitor"
+              },
+              "LogAnalyticsWorkspaceId": {
+                "value": "$LOG_ANALYTICS_WORKSPACE_ID"
+              },
+              "LogAnalyticsSharedKey": {
+                "value": "$LOG_ANALYTICS_SHARED_KEY"
+              },
+              "MetricsEndpointsCSV": {
+                "value": "http://edgeHub:9600/metrics,http://edgeAgent:9600/metrics,http://LoRaWanNetworkSrvModule:5000/metrics"
```

Comment:
Since we have the server pfx set here, will we not run into the issue of the self-signed certs and the metrics collector? (the LNS should listen on port 5001 in that case, no?)

Reply:
Very good point I removed it from here.